back-end/resources/room_config.json

```diff
@@ -21,44 +21,50 @@
       "output": {
         "greenLight1": {
           "type": "string",
```

Comment:
what does type do now?

Reply:
It specifies the type of that status output of that component